src/vs/workbench/browser/parts/editor/editorActions.ts

```diff
@@ -1553,3 +1553,75 @@ export class MoveEditorToNextGroupAction extends Action {
 		return TPromise.as(true);
 	}
 }
+
+export class MoveEditorToFirstGroupAction extends Action {
```

Comment:
@shobhitchittora all these actions are very similar in their implementation. Can we have a base class (abstract) that all classes extend from and in the base class we simply have a method to ask for the position?

Reply:
Yes on it.

Reply:
Done 💅🏻.
@bpasero Please review now.